### DISCUSSION TITLE
Cannot find module 'pretty-format' or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by michealroberts** [vitest-dev/vitest#421] [Q&A] [answered]
I have just integrated the latest version of vite within my project, and I am seeing the following issue:

```
Cannot find module 'pretty-format' or its corresponding type declarations.
```

**_Can anyone advise on why I am seeing this issue, and whether I should open up a bug?_**

Output of `npx envinfo --system --npmPackages '{vitest,vite,@vitejs/*}' --binaries --browsers`:

System:
    OS: macOS 12.0.1
    CPU: (8) arm64 Apple M1
    Memory: 135.08 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.13.0 - /usr/local/bin/node
    npm: 8.1.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 96.0.4664.110
    Firefox: 92.0
    Safari: 15.1
    Safari Technology Preview: 15.4
  npmPackages:
    vite: ^2.7.2 => 2.7.10 
    vitest: ^0.0.129 => 0.0.129

**Comment by sheremet-va** [comment, marked as answer]
This should be fixed in the latest release. Can you confirm?

**Reply by michealroberts** [reply]
@sheremet-va Hi Vladimir, yes this is all fixed in 0.0.130 now. Thank you to whoever patched this! Great stuff!
